website/client/src/components/ui/drawer.vue

```diff
@@ -77,17 +77,18 @@
     line-height: 1.67;
     color: $white;
     padding: 6px 0;
+    width: 408px;
```

Comment:
There were some changes made to this component in a different PR there are resulting in some conflicts, given that we're also using this component in other parts of the site it would be better to move these changes to spells.vue so that other drawers (see Inventory -> Equipment) are not affected